src/node.cc

```diff
@@ -4188,6 +4188,10 @@ void Init(int* argc,
   V8::SetFlagsFromString(NODE_V8_OPTIONS, sizeof(NODE_V8_OPTIONS) - 1);
 #endif
 
+  // Allow for environment set preserving symlinks
+  if (secure_getenv("NODE_PRESERVE_SYMLINKS")) {
+     config_preserve_symlinks = true;
+  }
```

Comment:
Please punctuate the comment and can you add a blank line after the block?

The way it's currently written means `env NODE_PRESERVE_SYMLINKS= node` will turn on the flag.  Most seasoned UNIX users probably won't expect that.